main/export.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const path = require('path');
+const {basename, parse} = require('path');
```

Comment:
Generally don't use destructuring for `path`. Most of the names are too generic without context and that makes the code harder to read. `parse` could mean so many things, but when you see `path.parse` it's much clearer.

Reply:
That makes a lot of sense 👍